### ISSUE TITLE
JBrowse-cli create command not working with last Jbrowse version

### ISSUE THREAD

**Issue by Akusem** [GMOD/jbrowse-components#1195]
**Describe the bug**
When using `jbrowse create ./jbrowse` command, I get the following error:
```
TypeError: Cannot read property 'browser_download_url' of undefined
```

After looking in `create.ts` it seem to be due to `fetchGithubVersions` not finding the release of Jbrowse.

I've tried it using differents tags, and only the latest version of the 3 releases avalaible (@gmod/jbrowse-web@0.0.1-beta.23) seem to have this problem, as, when using `jbrowse create ./jbrowse --tagVersions @gmod/jbrowse-web@0.0.1-beta.15` the installation seem to work correctly.

**To Reproduce**

1. Install last version of jbrowse-cli:  `yarn global add @gmod/jbrowse-cli@0.0.1-beta.21`
2. Try to create a jbrowse release: `jbrowse create ./jbrowse`

**Version:**
Jbrowse-cli 0.0.1-beta.21

**Comment by cmdcolin** [comment]
Thanks again for the report...

This is a knock on effect from this issue for automatically uploading our builds https://github.com/GMOD/jbrowse-components/issues/1184 (basically explains why we haven't been getting builds in things after beta.15) and I was futzing about more with it

I manually uploaded a build to the jbrowse-web@beta.23 tag so you can get a more recent release (this week)

Can also do

```
jbrowse create ./jbrowse --url https://s3.amazonaws.com/jbrowse.org/code/jb2/alpha/master/jbrowse-web-master.zip
```

To get our current master

**Comment by Akusem** [comment]
@cmdcolin I confirm it's working now, thanks for taking the time to manually upload the build.

**Comment by rbuels** [comment]
related to #1184

**Comment by cmdcolin** [comment]
Fixed by #1206
